### PR TITLE
Sequelize integration test, with sample package support page

### LINKED ISSUE #345: Issues with multi-database libs: sequelize
I am having trouble using any multi-database wrappers (tried sequelize and any-db) with ncc.

Let's take, for example, a URI like `sqlite:///tmp/demo.sqlite`, which works fine locally in an express service. If I compile with ncc, it runs fine if the output is in the same dir (where sqlite3 is installed.)

[Here](https://spectrum.chat/zeit/now/module-initialization-error-with-sequelize~727fd4ce-43f6-4544-8504-9c0a4889564a) is me troubleshooting with now.sh.

[Here](https://github.com/konsumer/demo-db) is a demo-repo that illustrates the problem. If I deploy or build locally but move the folder out of the project that has sqlite3 installed, it exhibits the problems above.

To reproduce:

```
git clone https://github.com/konsumer/demo-db.git
cd demo-db
npm i

ncc build node/index.js                                                              # builds fine
DATABASE_URI="sqlite:///tmp/demo.sqlite" node dist/index.js # runs fine

mv dist ~/Desktop
cd ~/Desktop/dist
DATABASE_URI="sqlite:///tmp/demo.sqlite" node index.js

/home/konsumer/Desktop/dist/index.js:28596
        throw new Error(`Please install ${moduleName} package manually`);
        ^

Error: Please install sqlite3 package manually
    at ConnectionManager._loadDialectModule (/home/konsumer/Desktop/dist/index.js:28596:15)
    at new ConnectionManager (/home/konsumer/Desktop/dist/index.js:21768:21)
    at new SqliteDialect (/home/konsumer/Desktop/dist/index.js:37324:30)
    at new Sequelize (/home/konsumer/Desktop/dist/index.js:22307:20)
    at Object.<anonymous> (/home/konsumer/Desktop/dist/index.js:72980:19)
    at __webpack_require__ (/home/konsumer/Desktop/dist/index.js:22:30)
    at startup (/home/konsumer/Desktop/dist/index.js:36:19)
    at /home/konsumer/Desktop/dist/index.js:42:18
    at Object.<anonymous> (/home/konsumer/Desktop/dist/index.js:45:10)
    at Module._compile (internal/modules/cjs/loader.js:805:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:816:10)
    at Module.load (internal/modules/cjs/loader.js:672:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:612:12)
    at Function.Module._load (internal/modules/cjs/loader.js:604:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:868:12)
    at internal/main/run_main_module.js:21:11

```

Might be related to #74

### LINKED ISSUE #397: Sequelize regression
On ncc 0.17.0-beta there is a Sequelize regression - 

```
/tmp/c39330374ab30c5bc8e34ebff9ef926e/index.js:48516
  const e = new Error('Cannot find module "' + id + '".');
            ^
Error: Cannot find module "./dialects//data-types".
    at Object.<anonymous> (/tmp/c39330374ab30c5bc8e34ebff9ef926e/index.js:48516:13)
    at __webpack_require__ (/webpack/bootstrap:19:1)
    at Object.module.exports.module.exports (/tmp/c39330374ab30c5bc8e34ebff9ef926e/index.js:37855:19)
    at __webpack_require__ (/webpack/bootstrap:19:1)
    at Object.<anonymous> (/tmp/node_modules/sequelize/lib/utils.js:3:19)
    at __webpack_require__ (/webpack/bootstrap:19:1)
    at Object.<anonymous> (/tmp/c39330374ab30c5bc8e34ebff9ef926e/index.js:49217:15)
    at __webpack_require__ (/webpack/bootstrap:19:1)
    at Object.module.exports.Object.defineProperty.value (/tmp/node_modules/sequelize/index.js:8:18)
    at __webpack_require__ (/webpack/bootstrap:19:1)
    at Object.<anonymous> (/tmp/test/integration/sequelize.js:1:19)
    at __webpack_require__ (/webpack/bootstrap:19:1)
    at startup (/webpack/bootstrap:33:1)
    at /webpack/bootstrap:39:1
    at Object.<anonymous> (/tmp/c39330374ab30c5bc8e34ebff9ef926e/index.js:45:10)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
```

### PR DESCRIPTION
**Pull request by guybedford**
This adds an integration test for sequelize, while also including a new documentation page linked to from the readme that we can use to document packages that need specific configurations for ncc.

In this case the `dialectModule` option for sequelize.

Resolves #397 and resolves #345.

### CODE REVIEW

**Comment by guybedford** [comment]
Note - this is passing with the asset relocator loader updates.

File: package-support.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+## ncc Package Configurations
+
+Packages that need specific configurations for ncc support are included below.
+
+If you are having trouble running a package with ncc, try searching for it in the [issue queue](https://github.com/zeit/ncc/issues) or posting an issue.
+
+### Sequelize
+
+For example, with the `maria-db`:
```
Comment (styfle) [inline package-support.md]:
```suggestion
For example, with the `mariadb` dialect:
```

**Comment by styfle** [comment]
In your comment from https://github.com/zeit/webpack-asset-relocator-loader/pull/39 it sounds like this fixes both sequelize and lighthouse.

Can you also add an integration test for lighthouse?

**Comment by guybedford** [comment]
Unfortunately there's another issue for Lighthouse - I've discussed this at https://github.com/zeit/ncc/issues/290#issuecomment-495177958.

**Comment by guybedford** [comment]
The coverage change seems like it might have been delayed from one of the previous PRs? Because the diff is the same otherwise.

**Comment by styfle** [comment]
@guybedford Ok I think this is good to merge.

I missed the last release but it looks like it was a beta. Why was that? And should the next release be beta too or stable?

**Pull request merged**